### PR TITLE
fix templating of team members, move them to data

### PR DESCRIPTION
**Pull request by jakubgs**
Changes:

* Drops obsolete `source/docs/team.md` which wasn't used anywhere
* Moves list of team members from `themes/navy/layout/index.swig` to `source/_data/team.yml`

### CODE REVIEW

File: source/_data/team.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- name: "Mamy Ratsimbazafy"
+  image: "/img/mamy.jpg"
+  statusLink: "https://join.status.im/user/0x04d01118c9cbcf4f9d326c1185bff110d36c96890a37cc47e7ce91e8b53ccc088d9df8c828c0669fb97831502113c1195bb17f1ea25fbccffee6a7fb9e30782a82"
+  statusName: "Damp Soulful Allensbigearedbat"
+  gitHubLink: "https://github.com/mratsim"
+
+- name: "Zahary Karadjov"
+  image: "/img/zahary.jpg"
+  statusLink: "https://join.status.im/user/0x049004fb151893f7112d754d92c650d6d35f8151ec0eff1c148812f8741474f7f8906436e782432680b40e96a2549c03d3dd7801c50928c2bf15d56f6d7d2e1246"
+  statusName: "Useless Gainsboro Serpent"
```
Comment (zah) [inline source/_data/team.yml]:
```suggestion
  statusName: "Nutritious Judicious Flicker"
```

That's my new name since v1.0

Comment (jakubgs) [reply source/_data/team.yml]: Nice thanks for the update.

File: source/_data/team.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  statusName: "Damp Soulful Allensbigearedbat"
+  gitHubLink: "https://github.com/mratsim"
+
+- name: "Zahary Karadjov"
+  image: "/img/zahary.jpg"
+  statusLink: "https://join.status.im/user/0x049004fb151893f7112d754d92c650d6d35f8151ec0eff1c148812f8741474f7f8906436e782432680b40e96a2549c03d3dd7801c50928c2bf15d56f6d7d2e1246"
+  statusName: "Nutritious Judicious Flicker"
+  gitHubLink: "https://github.com/zah"
+
+- name: "Yuriy Glukhov"
```
Comment (zah) [inline source/_data/team.yml]: We need to remove Yuriy and add @jangko, @dryajov, @sinkingsugar and @onqtam here.

Comment (jakubgs) [reply source/_data/team.yml]: Yeah, @arnetheduck said he will get the info for that, he just wanted me to make the template work properly.

File: themes/navy/layout/index.swig [hunk 3]
```diff
@@ -88,71 +88,16 @@
...
-                </p>
-            </div>
-            <div class="contribute">
-                <img class="contributor" src="../img/kdeme_moot.png" />
-                <h3>Kim De Mey</h3>
-                <p><a href="https://join.status.im/" style="color: #939ba1;"></a></p>
-                <p><a href="https://github.com/kdeme">See Github profile</a>
-                    <span><a href="https://github.com/kdeme"><img src="../img/arrow_ogn.png" /></a></span>
+                <img class="contributor" src="{{ member.image }}" />
+                <h3>{{ mamber.name }}</h3>
```
Comment (arnetheduck) [inline themes/navy/layout/index.swig]: mamber :)

Comment (jakubgs) [reply themes/navy/layout/index.swig]: :D

**Pull request merged**